generics/src/main/java/org/mentoring/exercise5/BadHabits.java

```diff
@@ -0,0 +1,12 @@
+package org.mentoring.exercise5;
+
+import java.util.List;
+
+public class BadHabits implements Habits{
```

Comment:
ctrl +alt + L w InteliJ formatuje kod. To bardzo ważne w programowaniu by klasy były identycznie sformatowane. Możesz poczytać też o pluginach typu Formatter tak na przyszłość.